docs/frequently-asked-questions.rst

```diff
@@ -431,7 +431,7 @@ What happens to a ``struct``'s mapping when copying over a ``struct``?
 This is a very interesting question. Suppose that we have a contract field set up like such::
 
     struct user {
-        mapping(string => address) usedContracts;
+        mapping(string => string) usedContracts;
```

Comment:
Would you mind changing the name `usedContracts` too as it doesn't seem to make much sense?